docs/README_API_INSTALL_DOCKER.md

```diff
@@ -1,13 +1,11 @@
 # README for Installation with Docker (OSX or Linux)
 [Back to root README](../README.md)
 
-**This installation method only requires Docker to be installed and running. All dependencies (libraries or Python modules) reside within the docker containers. These instructions are intended for advanced users familiar with docker, git and python.""
+Only [Docker Desktop](https://docs.docker.com/get-docker/) is required.
```

Comment:
For Linux, docker desktop is actually not required, just dockerd.

Reply:
That's true. I should have specified that this setup would be platform-agnostic, rather than just for Unix users. Windows users would need to get Docker Desktop.